src/engine/BMGame.php

```diff
@@ -437,17 +342,17 @@ protected function update_game_state_start_game() {
         $this->nRecentPasses = 0;
         $this->autopassArray = array_fill(0, $this->nPlayers, FALSE);
         $this->fireOvershootingArray = array_fill(0, $this->nPlayers, FALSE);
-        $this->gameScoreArrayArray = array_fill(0, $this->nPlayers, array(0, 0, 0));
```

Comment:
Actually, I think you have a good point here. If we're having to zero-fill, then we should actually be shifting these also into BMPlayer.